tests/expected/offset-from-bytes-overflow/main.rs

```diff
@@ -3,15 +3,25 @@
 
 // Check that an offset computed with `offset_from` triggers a verification failure
 // if it overflows an `isize` in bytes.
+
+// This example can be really confusing, but is the only way to reproduce the
+// failure we are looking for. This program works fine in Rust and it's even
+// possible to assert that the value coming from `offset_from` is equal to
+// `high_offset`. But CBMC's memory model is going to cause a "wrapping around"
+// behavior in `v_wrap`, so any values that depend on it are going to show a
+// strange behavior as well.
 use std::convert::TryInto;
 
 #[kani::proof]
 fn main() {
-    let v: &[u128] = &[0; 200];
-    let v_100: *const u128 = &v[100];
-    let max_offset = usize::MAX / std::mem::size_of::<u128>();
+    let v: &[u128] = &[0; 10];
+    let v_0: *const u128 = &v[0];
+    // `high_offset` is an offset that is high enough to:
+    //  * Not trigger failures in `v_0.add(...)`
+    //  * Trigger failures in `offset_from` after it
+    let high_offset = usize::MAX / (std::mem::size_of::<u128>() * 4);
     unsafe {
-        let v_wrap: *const u128 = v_100.add((max_offset + 1).try_into().unwrap());
-        let _ = v_wrap.offset_from(v_100) == 2;
+        let v_wrap: *const u128 = v_0.add(high_offset.try_into().unwrap());
```

Comment:
Add a comment here that explains this is UB, but Kani's default behavior doesn't catch it. We had to disable these checks because `std` does this kind of operation often.